beetsplug/edit.py

```diff
@@ -262,10 +291,20 @@ def edit_objects(self, objs, fields):
                         return False
 
                 # Show the changes.
+                # If the objects are not on the DB yet, we need a copy of their
+                # original state for show_model_changes.
+                if all(not obj.id for obj in objs):
```

Comment:
On a closer look, seems like a `deepcopy` of an object that **is** on the library might not be a good idea:
`TypeError: object.__new__(thread.lock) is not safe, use thread.lock.__new__()`